src/snarky.d.ts

```diff
@@ -417,6 +428,15 @@ declare class Bool {
   toBoolean(): boolean;
 
   /* static members */
+  /**
+   * The constant [[`Bool`]] that is `true`.
+   */
+  static true: Bool;
```

Comment:
I added `Bool.true` and `Bool.false`, because in the end it won the community vote :D

Reply:
Nice